frontend/src/components/sdcCollection/sdcSchoolCollection/StepFourDuplicatesProcessing.vue

```diff
@@ -295,6 +297,9 @@ export default {
           this.isLoading = false;
         });
     },
+    convertDate(dob) {
+      return LocalDate.parse(dob, DateTimeFormatter.ofPattern('uuuuMMdd').withResolverStyle(ResolverStyle.STRICT));
```

Comment:
We have a utils function to format dob. It's a bit different for yours, so ignore this comment if the other one wasn't working in this situation